content/post/apache-blog/success-at-apache-remote-collaboration.md

```diff
@@ -0,0 +1,92 @@
+---
+categories:
+- 开源
+- Apache way
+- 博客
+- 传播
+date: 2020-05-25T16:06:05+08:00
+description: "Apache 之道的异步通信，是否在疫情发生的非常时期能够凸显出其优势了呢？还是只是没有被影响到罢了？让我们来听听 ASF 的名誉会员怎么讲。"
+keywords:
+- Open Source
+- Culture
+- Reading
+- News
+- Blog
+tags:
+- Apache blog
+title: "【Apache 博客】之 冠状病毒肆虐之下的远程协作"
+url: ""
+authors:
+- Marvin Humphrey
+translater:
+- ALC Beijing
+---
+
+## ALC Beijing 引言
+
+2020年，对于世界上所有的人都是一场灾难，在刚刚开完的两会，国家希望通过地摊经济来复苏中华大地的经济繁荣，当然这在日常的小商品、食物等方面还可以，那么信息经济下，过去一直都在一个屋檐下工作的人，突然变成了在线会议的方式，一定会出现一定不适应，甚至会出现效率低下。这个时候，最应该考虑的就是开源的协作模式！当然，ASF 以其独特的完全虚拟享誉全球！值得学习的地方实在是太多了。
+
+![](https://codeahoy.com/assets/images/featured/corona_virus.jpg)
+
+## 前序
+
+我（Marvin Humphrey）是在2005年”抵达“ ASF （Apache 软件基金会的简称，下同） 的，起因是我对 Apache Lucene 的一个Bug有了严重的不满情绪，之所以使用“抵达”这个词汇，就像是我找到了某种归属，因为自从那时发起了第一封邮件之后，在接下来的15年里，我发送了数以万计的邮件，可想而知我在ASF花了多长时间，这几乎是我的第二个家。由于 ASF 一直以来都没有实际的“物理”办公室，所以也就免去了很多现实中的麻烦，比如是否穿戴得体、肢体语言是否合规等等，其实就那个Bug 来讲，我当时不满的原因是：Doug Cutting （Hadoop创始人 ——译者注），当时还是Lucene 项目的贡献者，认为他非常没有耐心。
+
+让我们回到本文的主题上了，因为Apache 这20多年来没有实际的办公室，大家均是基于互联网这个虚拟的线上共同体而工作在一起，所以新冠病毒发生以来，其实对于 Apache 的成员和贡献者们的影响是微乎其微的，尽管对于每位贡献者的个人生活上是确确实实受到了影响，但是就ASF的整体而言，并没有出现什么不适应之类的。
+
+但是 ASF 之外仍然有很多的人，他们未必有这样的一套机制和习惯，世界各地的人们都在为抗争疫情而努力，尤其是“社交距离”的约定，原有的公司章程等均可为此而进行修订，另外让人痛心不已的是，有些人根本都没有机会来进行改变，直接破产倒闭，人已经失业了。
+
+然而，决定改变的人们遭遇到了一些窘境，是将大量的工作采用了远程的方式来做，但是进展缓慢；突然而至的远程工作，人们措手不及。那么我们是不是可以如此的积极思考了呢？加快并简化向远程协作的过渡，那么可以节省不少的精力，并可以提高效率。这里可能需要加上一点创造力，面对疫情之下的“新常态”，是可以做到比原来更加高效的，而且也不需要花多久就可以适应。
+
+请问你是开源贡献者吗？如果是的话，那么贡献你，你已经具备了当前全球复杂经济面前的急需的远程工作的能力和知识，接下来就让我们来聊聊我们所知道的以及或许可以帮助到你的一些内容。
+
+## 互联网将人们变成了“怪物”
+
+在互联网，人们彼此的交流大多是通过对着电脑屏幕敲打键盘完成的，这区别于我们现实中面对面的语言交流。作为社会性的人类，我们可以通过他人的面部表情、语气、肢体语言等反馈来对自己进行相应的校准，并作出适当的反应。当时，当所有的沟通和交流都通过击打键盘来完成的时候，就没有了面对面那种反馈机制，这对于人来说是及其难以适应的事情，往往会产生一些不好的效果，尤其是带有情绪的词汇。
+
+那么，过往一直以来都是在办公室交流的人们，突然转移到了Slack、电子邮件、以及其它以文本的方式交流的方式和渠道，有一定的不适应是较为正常的，但是通过文本来进行沟通是有难度的，需要训练自身的同理心来获得提升，个人建议去观看“Jimmy Kimmel Show” 当中的[“名人阅读Tweets”](https://www.youtube.com/playlist?list=PLs4hTtftqnlAkiQNdWn6bbKUr-P1wuSm0)节目, 可以有效提升自己对于一些词语的敏感度。
+
+世界是动态演变的，只要开源共同体仍然有存在的意义，那么它就必须在每一天都要和这些不便进行斗争。随着时间的流逝，“阴谋”的人身攻击由于其潜在的腐蚀作用而逐渐成为禁忌。这就成就了我们在文化上的共识：**应该去评判那些想法，而不是人本身。**
+
+随着越来越多的共同体采用“行为准则”，辩护变得越来越正式和复杂。这些明文的文档，主要目的是防止骚扰，以及其它严重的不当行为，这些文档通常包含有关共同体成员应如何彼此相待的颇为理想的建议，在鱼龙混杂的环境下特别容易产生一些严重的不当行为。
```

Comment:
While the primary purpose of such documents is guard aginst harassment and other serious misconduct, they often contain aspirational recommendations about how community members should treat each other — because serious misconduct is more likely to occur in an environment of constant low-grade incivility.

最后一句与原文还是有些差距， 建议在”在鱼龙混杂的环境”之前加上“因为”。